lib/services/auth_service.dart

```diff
@@ -60,7 +61,8 @@ class AuthService {
           user.serverAdress!, user.name!, user.password!);
     } catch (e) {
       if (user.serverAdress!.split(":").last != "8096" &&
-          user.serverAdress!.split(":").length == 2) {
+          user.serverAdress!.split(":").length == 2 &&
```

Comment:
this will not work. what if my server is at http://[2001:0db8:85a3:0000:0000:8a2e:0370:7334]:8096
its true that the previous code was broken in this way as well, but not taking this opportunity to fix it would be silly.  are you completely against using the regex I linked you?  its really one of very few correct ways of approaching this issue.

Reply:
I strongly recommend following an approach like this (but avoiding the brightscriptisms, obvi):
https://github.com/jellyfin/jellyfin-roku/blob/4e872f42936e02ca2134933dc1b3186d219c14e9/source/utils/misc.bs#L185
https://github.com/jellyfin/jellyfin-roku/blob/4e872f42936e02ca2134933dc1b3186d219c14e9/source/utils/misc.bs#L238
using this regex:
https://github.com/jellyfin/jellyfin-roku/blob/4e872f42936e02ca2134933dc1b3186d219c14e9/source/utils/misc.bs#L364